sdk/python/tests/integration/registration/test_cli_apply_duplicates.py

```diff
@@ -2,16 +2,20 @@
 from pathlib import Path
 from textwrap import dedent
 
+import pytest
+
 from tests.utils.cli_utils import CliRunner, get_example_repo
 
 
+@pytest.mark.integration
 def test_cli_apply_duplicated_featureview_names() -> None:
     run_simple_apply_test(
         example_repo_file_name="example_feature_repo_with_duplicated_featureview_names.py",
         expected_error=b"Please ensure that all feature view names are case-insensitively unique",
     )
 
 
+@pytest.mark.integration
 def test_cli_apply_duplicate_data_source_names() -> None:
     run_simple_apply_test(
         example_repo_file_name="example_repo_duplicate_data_source_names.py",
```

Comment:
I'll actually argue that a test that can run without any network connectivity (cause it uses file source and sqlite online store) should be a unit test.

Unit tests are more reliable and run faster, and we wanna make more tests unit tests anyway

Reply:
Applies to the other `test_cli_` tests too.

Reply:
Did major refactor for those